canonical-web-and-design/ubuntu.com
# Add redirect to k8s survey

## Done

- Add redirect from  https://ubuntu.com/blog/85-of-enterprises-have-yet-to-cross-the-chasm-to-full-kubernetes-and-cloud-native-adoption-according-to-industry-global-survey-from-canonical to https://ubuntu.com/blog/canonical-global-survey-kubernetes-cloud-native



## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- *Turn on your VPN*
- View the site locally in your web browser at: http://0.0.0.0:8001/blog/85-of-enterprises-have-yet-to-cross-the-chasm-to-full-kubernetes-and-cloud-native-adoption-according-to-industry-global-survey-from-canonical
- See that you end up on http://0.0.0.0:8001/blog/canonical-global-survey-kubernetes-cloud-native
